From 1b69a97c1dddaa9fca57f99ed90000d876d8b42c Mon Sep 17 00:00:00 2001
From: Miaoqing Pan <quic_miaoqing@quicinc.com>
Date: Wed, 30 Nov 2022 10:50:59 +0800
Subject: [PATCH] libnl-tiny: fix redefinition of 'struct ifmap'

'struct ifmap' is also defined in 'net/if.h', remove the defination
from libnl-tiny.

Signed-off-by: Miaoqing Pan <quic_miaoqing@quicinc.com>
---
 include/linux/if.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/if.h b/include/linux/if.h
index 4c1bcfe..724b1d0 100644
--- a/include/linux/if.h
+++ b/include/linux/if.h
@@ -116,6 +116,7 @@ enum {
  *	being very small might be worth keeping for clean configuration.
  */
 
+#if 0
 struct ifmap 
 {
 	unsigned long mem_start;
@@ -126,6 +127,7 @@ struct ifmap
 	unsigned char port;
 	/* 3 bytes spare */
 };
+#endif
 
 
 #endif /* _LINUX_IF_H */
-- 
2.7.4

